src/tkn.ts

```diff
@@ -876,6 +902,7 @@ export interface Tkn {
   getPipelineRunsList(pipelineRun?: TektonNode): Promise<TektonNode[]>;
   getTaskRunList(taskRun?: TektonNode): Promise<TektonNode[]>;
   getRawPipelineRun(name: string): Promise<PipelineRunData | undefined>;
+  _getPipelineRuns(pipeline: TektonNode, pipelineName?: string): Promise<TektonNode[]> | undefined;
```

Comment:
Why do you have `_` in function name? Maybe better to find more descriptive name for that function.